src/v2/Apps/Unsubscribe/UnsubscribeApp.tsx

```diff
@@ -0,0 +1,58 @@
+import { Column, GridColumns, Spacer, Text, Message } from "@artsy/palette"
+import React from "react"
+import { Title } from "react-head"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useRouter } from "v2/Artsy/Router/useRouter"
+import { UnsubscribeApp_me } from "v2/__generated__/UnsubscribeApp_me.graphql"
+import { UnsubscribeLoggedInFragmentContainer } from "./Components/UnsubscribeLoggedIn"
+import { UnsubscribeLoggedOut } from "./Components/UnsubscribeLoggedOut"
+
+interface UnsubscribeAppProps {
+  me: UnsubscribeApp_me | null
+}
+
+export const UnsubscribeApp: React.FC<UnsubscribeAppProps> = ({ me }) => {
+  const { match } = useRouter()
+  const { authentication_token: authenticationToken } = match.location.query
+
+  if (!me && !authenticationToken) {
+    return (
+      <Message variant="error" my={4}>
+        Unable to update your email preferences
+      </Message>
+    )
+  }
+
+  return (
+    <>
+      <Title>Email Preferences | Artsy</Title>
+
+      <GridColumns my={4}>
+        <Column span={6}>
+          <Text variant="xl" as="h1">
+            Email Preferences
+          </Text>
+
+          <Spacer mt={6} />
+
+          {!me ? (
+            <UnsubscribeLoggedOut authenticationToken={authenticationToken} />
+          ) : (
+            <UnsubscribeLoggedInFragmentContainer me={me} />
+          )}
```

Comment:
I have a slight preference for writing ternary expressions in the affirmative. No biggie but noticed it here.

```suggestion
          {me ? (
            <UnsubscribeLoggedInFragmentContainer me={me} />
          ) : (
            <UnsubscribeLoggedOut authenticationToken={authenticationToken} />
          )}
```